bridge/opencensus/internal/otel2oc/span_context_test.go

```diff
@@ -4,14 +4,26 @@
 package otel2oc
 
 import (
+	"strings"
 	"testing"
 
+	"go.opencensus.io/trace/tracestate"
+
 	octrace "go.opencensus.io/trace"
 
 	"go.opentelemetry.io/otel/trace"
 )
 
 func TestSpanContextConversion(t *testing.T) {
+	tsOc, _ := tracestate.New(nil,
+		// Oc has a reverse order of TraceState entries compared to OTel
+		tracestate.Entry{Key: "key2", Value: "value2"},
+		tracestate.Entry{Key: "key1", Value: "value1"},
```

Comment:
We should have the same order.

From https://github.com/open-telemetry/opentelemetry-go/pull/5651#discussion_r1696503632:

> If the consumer does not like the ordering then it can reverse it by himself.

Reply:
In OC `TraceState.Entries()`, the returned entries is in the order of insertion order.

Reply:
> In OC `TraceState.Entries()`, the returned entries is in the order of insertion order. What's your suggestion to fix this?

This is not really true. See https://pkg.go.dev/go.opencensus.io@v0.24.0/trace/tracestate#New (and look at the implementation).

Reply:
Based on https://github.com/census-instrumentation/opencensus-go/blob/01e6da5fc01c42aca1e0ce315f41744876e9fcbb/trace/tracestate/tracestate.go#L84, it will keep the original order of entries. but they will be inserted before parent tracestate. And if what you said is true, my test won't pass. Am I missing something?

Reply:
No. It is me who got confused. I think everything is fine 👍